tests/scripts/combined-coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+npm run test:unit
+npm run test:visual
+
+# merge
+mkdir -p coverage/merge
+cp coverage/unit/coverage-final.json coverage/merge/coverage-unit.json
+cp coverage/visual/coverage-storybook.json coverage/merge/coverage-storybook.json
+
+nyc report --reporter=lcov --reporter=text -t coverage/merge --report-dir coverage/merge
+cp coverage/merge/lcov.info coverage/lcov.info
```

Comment:
to check my understanding, the coverage check will fail if the combined coverage drops, right? or is it just if the unit test coverage drops?

Reply:
the coverage check not uses the combined coverage file (I need to copy the combined lcov.info to top level because it's where the coverage github action expects it to be right now)